java/java.lsp.server/nbcode/integration/src/org/netbeans/modules/nbcode/integration/resources/uimanager-icons.properties

```diff
@@ -0,0 +1,63 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This file lists icons to be replaced by URL-loaded images. 
+# The format is j.u.Properties; if value is missing, the key will be prefixed
+# with org/netbeans/modules/nbcode/integration/resources/uidefaults/, suffix
+# .png. If the value is present, it must be the full path to the resource incl.
+# extension, that can be loaded by ImageUtilities.loadImage.
+#
+# Buildscript will automatically create stub resources for keys without value.
+
+CheckBox.icon
+ComboBox.icon
+ETableHeader.ascendingIcon
+ETableHeader.descendingIcon
+FileChooser.homeFolderIcon
+FileChooser.newFolderIcon
+FileChooser.upFolderIcon
+Menu.arrowIcon
+MSG_WaitScan
+nb.bigclose.tab.icon.enabled.icon
+nb.bigclose.tab.icon.pressed.icon
+nb.bigclose.tab.icon.rollover.icon
+nb.close.tab.icon.enabled.icon
+nb.close.tab.icon.pressed.icon
+nb.close.tab.icon.rollover.icon
+nb.multitabs.button.dropdown.icon
+nb.multitabs.button.left.icon
+nb.multitabs.button.maximize.icon
+nb.multitabs.button.right.icon
+nb.popupswitcher.closebutton.defaultIcon
+nb.popupswitcher.closebutton.rolloverIcon
+nb.tabcontrol.busy.icon.normal
+nb.tabcontrol.busy.icon.selected
+OptionPane.informationIcon
+OptionPane.warningIcon
+PropSheet.customButtonIcon
+RadioButton.icon
+Table.columnSelection
+ToolBar.handleIcon
+Nb.Explorer.Folder.icon
+Nb.Explorer.Folder.openedIcon
+Tree.gtk_collapsedIcon
+Tree.gtk_expandedIcon
+Tree.closedIcon=org/netbeans/modules/nbcode/integration/resources/uidefaults/Tree.closedIcon.png
```

Comment:
Will this new png file show up somewhere in the GUI? There are better open/closed folder icons available in the codebase (and even SVG versions)... but I assume this is just a placeholder default.

Reply:
They might show. Pls. suggest the more appropriate ones - I'll change them when I incorporate SVG support into the LSP protocol. From the (current) vscode release's (12.6.301, 13.0) perspective, these are just placeholders.

Reply:
Looking at the icon resource files, I think these are the best PNG files to use; the new ones are SVG files only. You can include apisupport/apisupport.ant/src/org/netbeans/modules/apisupport/project/resources/defaultFolder.svg and
apisupport/apisupport.ant/src/org/netbeans/modules/apisupport/project/resources/defaultFolderOpen.svg alongside the old PNG files for when you want to support SVG support.